cores/arduino/am_sdk_ap3/mcu/apollo3/hal/am_hal_uart.h

```diff
@@ -172,8 +172,8 @@ am_hal_uart_transfer_t;
 
```

Comment:
How come we had to modify the HAL layer?
Some sort of compiler error? (By looking at it and being familiar with the HAL I would guess that there was no definition for ```UART_CR_CTSEN_Msk``` and that they did intend to use the UART0 macro as the base no matter which UART you are actually working on.)
If that is true we should be documenting changes to the HAL in this repo:https://github.com/sparkfun/AmbiqSuiteSDK

Reply:
Yes, not defined.

I've created PR over on HAL repo.